src/scripts/ci/travis.yml

```diff
@@ -11,9 +11,13 @@ compiler:
   - clang
   - gcc
 
+services:
+  - docker
+
 env:
   global:
     - CCACHE_MAXSIZE=800M
+    - SDK=21
```

Comment:
For just invoking the script on the side a short var name like `SDK` was fine but setting it here it is not obvious what it controls. Rename to `ANDROID_SDK_VER` or something.